cmd/in-toto/root.go

```diff
@@ -27,6 +28,15 @@ var rootCmd = &cobra.Command{
 	SilenceErrors: true,
 }
 
+func init() {
+	rootCmd.PersistentFlags().StringVar(
+		&spiffeUDS,
+		"spiffe-workload-api-path",
+		"",
+		"uds path for spiffe workload api",
+	)
+}
+
```

Comment:
I think we should move this to the individual commands where it's used, even if it means copying the same line a few times, to avoid confusing people with a flag that does nothing on some commands.

Reply:
Moved to where it is being used instead of global.